charts/templates/_helpers.tpl

```diff
@@ -129,16 +129,24 @@ Create a default fully qualified app name.
 We truncate at 63 chars because some Kubernetes name fields are limited to this (by the DNS naming spec).
 */}}
 {{- define "clusterpedia.postgresql.fullname" -}}
+{{- if contains "postgresql" .Release.Name -}}
+{{- .Release.Name | trunc 63 | trimSuffix "-" -}}
```

Comment:
This does fix https://github.com/clusterpedia-io/clusterpedia/issues/121.

Thinking further, the problem may still occur when `--set postgresql.fullnameOverride=<some>` is set, so we can refer to the [fullname template](https://github.com/bitnami/charts/blob/6a0f3ad9153a79d7f6e66d50494d741d870b9383/bitnami/common/templates/_names.tpl#L21) used by bitnami/postgresql.

Suggestion：
```
{{- define "clusterpedia.postgresql.fullname" -}}
{{- if .Values.postgresql.fullnameOverride -}}
{{- .Values.postgresql.fullnameOverride | trunc 63 | trimSuffix "-" -}}
{{- else -}}
{{- $name := default "postgresql" .Values.postgresql.nameOverride -}}
{{- if contains $name .Release.Name -}}
{{- .Release.Name | trunc 63 | trimSuffix "-" -}}
{{- else -}}
{{- printf "%s-%s" .Release.Name $name | trunc 63 | trimSuffix "-" -}}
{{- end -}}
{{- end -}}
{{- end -}}
```

Reply:
Sure, thanks for your suggestion.